.github/scripts/push-docs.sh

```diff
@@ -18,7 +18,7 @@ fi
 # Do not remove the trailing period!!!
 # https://dev.to/ackshaey/macos-vs-linux-the-cp-command-will-trip-you-up-2p00
 INPUT_SOURCE_FOLDER="./docs/."
-INPUT_DESTINATION_REPO="immutable/imx-docs"
+INPUT_DESTINATION_REPO="immutable/docs"
```

Comment:
I think you might also want to check .github/workflows/publish-docs.yaml

Reply:
Thanks. Fixed.